kernel/src/hil/digest.rs

```diff
@@ -41,6 +76,23 @@ pub trait Digest<'a, const L: usize> {
         data: LeasableBuffer<'static, u8>,
     ) -> Result<usize, (ErrorCode, &'static mut [u8])>;
 
+    /// Clear the keys and any other sensitive data.
```

Comment:
Where are the keys set? For a digest there shouldn't be keys?

Reply:
The keys are set in the `set_mode*()` functions